src/mocking/server/data.ts

```diff
@@ -16,6 +16,12 @@ import {
 type MirageJSServer = any
 
 export const createMockData = (server: MirageJSServer) => {
+  mockChannels.forEach((channel, idx) => {
```

Comment:
Small thing, but maybe we can do that in `mockChannels.map` below, as part of the same iteration? Like we do for `mockVideos`